docs/tutorial.rst

```diff
@@ -7,7 +7,9 @@ This tutorial will walk you through installing, configuring, and running Nox.
 Installation
 ------------
 
-Nox can be easily installed via `pip`_::
+Nox can be easily installed via `pip`_:
+
+.. code-block:: console
```

Comment:
**ASIDE**: It would have been nice to have all of these `.. code-block` changes in a separate commit